### PR TITLE
Tools cutoff

### PR DESCRIPTION
**Pull request by rannick**
Closes https://github.com/nf-core/rnafusion/issues/354 and https://github.com/nf-core/rnafusion/issues/367

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnafusion/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, also make a PR on the nf-core/rnafusion _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -245,25 +249,15 @@
...
                 },
-                "whitelist": {
-                    "type": "string",
-                    "fa_icon": "far fa-file-code",
-                    "description": "Path to fusions to add to the input of fusioninspector"
-                },
-                "cosmic_username": {
-                    "type": "string",
+                "tools_cutoff": {
+                    "type": "integer",
```
Comment (fevac) [inline nextflow_schema.json:253]: Do we need a validator here? e.g. what happens when --tools_cutoff is 0 ?

File: subworkflows/local/fusioninspector_workflow.nf [hunk 2]
```diff
@@ -17,19 +17,18 @@ workflow FUSIONINSPECTOR_WORKFLOW {
     main:
         ch_versions = Channel.empty()
         index ="${params.starfusion_ref}"
-        ch_fusion_list = params.fusioninspector_filter ? fusion_list_filtered : fusion_list
 
         if (params.whitelist)  {
-            ch_whitelist = ch_fusion_list.combine(Channel.value(file(params.whitelist, checkIfExists:true)))
+            ch_whitelist = fusion_list_filtered.combine(Channel.value(file(params.whitelist, checkIfExists:true)))
```
Comment (fevac) [inline subworkflows/local/fusioninspector_workflow.nf:22]: I wonder about the naming convention, sometimes channels seem to start by `ch` sometimes they don't. Should this be `ch_fusion_list_filtered` ?

**Review by fevac — CHANGES_REQUESTED** [review]
Nice work! 🌟

**Comment by fevac** [comment]
to me this might be part of a major update since you're making incompatible changes. Maybe it's better to leave for next major release?

**Comment by rannick** [comment]
Yep let's wait and merge for the 3.0.0 release

File: subworkflows/local/fusioninspector_workflow.nf (hunk 2 above)
Comment (rannick) [reply subworkflows/local/fusioninspector_workflow.nf]: So I left it like this for the moment and opened: https://github.com/nf-core/rnafusion/issues/392 to harmonize in one go

File: nextflow_schema.json (hunk 1 above)
Comment (rannick) [reply nextflow_schema.json:253]: Resolved in [491501c](https://github.com/nf-core/rnafusion/pull/389/commits/491501c9f6ac97f518ab4a46034581869e8546aa)

**Pull request merged**